app/views/api/current/analytics/user_analytics/show.html.erb

```diff
@@ -17,18 +17,18 @@
 <% else %>
   <div class="card">
     <%= inline_svg('icon_new_bp.svg', classname: 'icon-float') %>
-    <h3>How many patients have a BP recorded at <%= @current_facility.name %>?</h3>
+    <h3>How many follow-up patients visited <%= @current_facility.name %>?</h3>
```

Comment:
This string should be internationalized.

Reply:
Lets pick this up in the transifex PR